pkg/controller/plan/validation.go

```diff
@@ -792,14 +814,37 @@ func (r *Reconciler) validateVddkImage(plan *api.Plan) (err error) {
 			r.Log.Info("validation of VDDK job is in progress", "image", image)
 			plan.Status.SetCondition(vddkValidationInProgress)
 		}
+		pods := &core.PodList{}
+		listerr := r.List(context.TODO(), pods, &client.ListOptions{
```

Comment:
note that the validation job may run on a separate k8s/OpenShift cluster and in that case, we need to use a client to the "destination cluster" (see use of `ctx.Destination.Client` below) - I think that this way we use the client to the cluster we run on